index.php

```diff
@@ -150,6 +150,49 @@ function gtag(){dataLayer.push(arguments);}
                     </div>
                   </div>
                 </div>
+
+                <div class="<?php echo (isset($_GET['eip1559']) ? "show" : "hidden") ?>">
+                  <div class="top_tiles_title">
+                    Recommended (<a href="https://ethgasstation.info/blog/eip-1559/">EIP-1559</a>) tipping
+                  </div>
+
+                  <div class="col-md-4 col-sm-4 col-xs-6 tile_stats_count">
+                    <div>
+                      <div class="count fast">
+                        <?php echo ($gpTip['instant']) ?>
```

Comment:
Also verified, if one of the key doesn't exist (which shouldn't happen) or if there's a typo in the variable name itself, the result is simply empty, but the application doesn't crash.
So we're safe here too